adal/build.gradle

```diff
@@ -78,6 +79,45 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+
+task createPom  {
+    pom {
+        project {
+            groupId 'com.microsoft.aad'
+            artifactId 'adal'
+            version = project.version
+            packaging 'aar'
+            name 'adal'
+
+            description 'Azure active directory library for Android gives you the ability to add Windows Azure Active Directory authentication to your application with just a few lines of additional code. Using our ADAL SDKs you can quickly and easily extend your existing application to all the employees that use Windows Azure AD and Active Directory on-premises using Active Directory Federation Services, including Office365 customers.'
+            url 'https://github.com/MSOpenTech/azure-activedirectory-library-for-android'
+
+            developers {
+                developer {
+                    id 'msopentech'
+                    name 'Microsoft Open Technologies, Inc.'
```

Comment:
@heidijinxujia Got it. We need to change this.  I'm not sure what to though. Perhaps we should look at the Azure SDK for inspiration.